### ISSUE TITLE
Reconsider decimal places in baseline adjustment

### ISSUE THREAD

**Issue by DCEW** [The-Strategy-Unit/nhp_inputs#704]
Currently users are shown the relative change to 6 decimal places - seems excessive, and users have questioned whether this is needed.

I'd suggest 2 or 3 would be sufficient? Any reason why we have so many?

**Comment by matt-dray** [comment]
Don't think there's a particular reason. Probably a missing `round()`.
